changelogs/unreleased/4410-installation-constraints.yml

```diff
@@ -0,0 +1,6 @@
+description: Take into account the requirements.txt on the project when installing or updating
+issue-nr: 4410
+change-type: patch
+destination-branches: [master, iso5]
+sections:
+    upgrade-note: "`inmanta project install` and `inmanta project update` now always take into account the `requirements.txt` of the project to provide additional version constraints to pip"
```

Comment:
Is this OK? I'm not too happy about the formulation?

Reply:
Looks good to me. I don't have any suggestions for a better formulation.